apps/site/assets/ts/stop/components/sidebar/Accessibility.tsx

```diff
@@ -40,12 +40,7 @@ const accessibilityKnown = (stop: Stop): boolean =>
   !stop.accessibility.includes("unknown");
 
 const isSpecialCase = (stop: Stop): boolean => {
-  if (
-    ["place-lech", "place-brkhl", "place-newtn", "place-asmnl"].includes(
-      stop.id
-    )
-  )
-    return true;
+  if (["place-asmnl"].includes(stop.id)) return true;
```

Comment:
this function can be simplified to 
`const isSpecialCase = (stop: Stop): boolean => stop.id === "place-asmnl";`